packages/components/organisms/f-checkout/src/store/checkout.module.js

```diff
@@ -355,7 +355,7 @@ export default {
 
             if (isAddressInLocalStorage) {
                 const storedAddress = addressService.getAddressFromLocalStorage(false);
-                if (storedAddress.Line1 === state.address.line1 && storedAddress.PostalCode === state.address.postcode) {
+                if (addressService.doesAddressInStorageAndFormMatch(storedAddress, state.address)) {
```

Comment:
I used this with both addresses passed in as we don't want to be reading from local storage repeatedly. But I could check if they match as part of the check to see if the address is in local storage?